packages/curve-ui-kit/src/themes/design/1_sizes_spaces.ts

```diff
@@ -306,4 +306,5 @@ export const SizesAndSpaces = {
     minHeight: '300px',
   },
   OutlineWidth: '0.125rem', // 2px
+  FooterWidth: '96rem', // 1536px
```

Comment:
we should make the header consistent too. I used the [default mui `<Container>`](https://github.com/curvefi/curve-frontend/blob/5545c375e9bcf0f5d46c39d7645728a90b59d698/packages/curve-common/src/widgets/Header/DesktopHeader.tsx#L37) which [follows `var(--width)`](https://github.com/curvefi/curve-frontend/blob/da6bf3f7d877cb6357e0c2a5c6819f76c46247cb/packages/curve-ui-kit/src/themes/components.ts#L40), currently mapped to [1440px](https://github.com/curvefi/curve-frontend/blob/9caa81d15b97dc4486d42e340c0a9674967713bb/packages/ui/src/styles/layout.css#L24)

Reply:
as discussed with @0xtutti we should update `--width` to 96rem